corehq/apps/locations/static/locations/js/import.js

```diff
@@ -2,7 +2,6 @@ hqDefine('locations/js/import', [
     'jquery',
     'knockout',
     'analytix/js/google',
-    'commtrack/js/location_bulk_upload_file',
```

Comment:
Stumbled on this: this file was deleted but this line wasn't removed, it's currently throwing a 404.